genesyscloud/util/util_strings.go

```diff
@@ -9,6 +9,10 @@ var matchFirstCap = regexp.MustCompile("(.)([A-Z][a-z]+)")
 var matchAllCap = regexp.MustCompile("([a-z0-9])([A-Z])")
 var matchUnderscore = regexp.MustCompile("_")
 
+func ToStrPtr(s string) *string {
```

Comment:
Useful little method I wrote to convert a string to *String pointer.  That way you don't have to assign a String to a variable just to pass it into a struct

Reply:
We already have this function in the go sdk, `platformclientv2.String()`. There's also one's for integers, booleans and floats

Reply:
Nice.  I did not know this.  Removing this function and replacing ToPtrStr with the platform call